NeuralAmpModeler/NeuralAmpModeler.cpp

```diff
@@ -60,7 +60,7 @@ NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo& info)
   activations::Activation::enable_fast_tanh();
   GetParam(kInputLevel)->InitGain("Input", 0.0, -20.0, 20.0, 0.1);
   GetParam(kToneBass)->InitDouble("Bass", 5.0, 0.0, 10.0, 0.1);
-  GetParam(kToneMid)->InitDouble("Middle", 5.0, 0.0, 10.0, 0.1);
+  GetParam(kToneMid)->InitDouble("Mid", 5.0, 0.0, 10.0, 0.1);
```

Comment:
Would this mess with people's saved presets?

Reply:
no